lib/workers/repository/process/lookup/index.ts

```diff
@@ -372,6 +372,45 @@ export async function lookupUpdates(
       );
       let shrinkedViaVulnerability = false;
       if (config.isVulnerabilityAlert) {
+        if (config.vulnerabilityFixVersion) {
+          res.vulnerabilityFixVersion = config.vulnerabilityFixVersion;
+          if (versioning.isValid(config.vulnerabilityFixVersion)) {
+            // Filter out versions if the vulnerabilityFixVersion is higher
+            const filterCondition = (release: Release): boolean =>
```

Comment:
this inline function is hard to read, can you move it to module scope and add a tsdoc comment there?

Reply:
Refactored a different way